pkg/webhooks/server.go

```diff
@@ -234,30 +236,36 @@ func (ws *WebhookServer) handleMutateAdmissionRequest(request *v1beta1.Admission
 			},
 		}
 	}
-
-	// MUTATION
-	// mutation failure should not block the resource creation
-	// any mutation failure is reported as the violation
-	patches := ws.HandleMutation(request, resource, policies, roles, clusterRoles)
-
-	// patch the resource with patches before handling validation rules
-	patchedResource := processResourceWithPatches(patches, request.Object.Raw, logger)
-
-	if ws.resourceWebhookWatcher != nil && ws.resourceWebhookWatcher.RunValidationInMutatingWebhook == "true" {
-		// VALIDATION
-		ok, msg := ws.HandleValidation(request, policies, patchedResource, roles, clusterRoles)
-		if !ok {
-			logger.Info("admission request denied")
-			return &v1beta1.AdmissionResponse{
-				Allowed: false,
-				Result: &metav1.Status{
-					Status:  "Failure",
-					Message: msg,
-				},
+	var patchedResource []byte
```

Comment:
We don't want to set empty byte as default value - If `!versionCheck`, it would pass empty byte to `HandleGenerate`.